tools/istio-nftables/pkg/capture/run.go

```
@@ -0,0 +1,1026 @@
+type NftablesConfigurator struct {
+	cfg              *config.Config
+	NetworkNamespace string
+	ruleBuilder      *builder.NftablesRuleBuilder // Helper to construct nftables rules
+	nftProvider      func(family knftables.Family, table string) (NftablesAPI, error)
+}
+
+// NetworkRange is used to hold IPv4 or IPv6 ranges.
+type NetworkRange struct {
+	IsWildcard    bool
+	CIDRs         []netip.Prefix
+	HasLoopBackIP bool
+}
+
+func split(s string) []string {
+	return config.Split(s)
+}
+
+// NewNftablesConfigurator initializes a new configurator instance.
+// If nftProvider is not supplied, it uses the real system provider with the default inet family.
+func NewNftablesConfigurator(cfg *config.Config, nftProvider func(family knftables.Family, table string) (NftablesAPI, error)) (*NftablesConfigurator, error) {
+	if cfg == nil {
+		cfg = &config.Config{}
+	}
+
+	if nftProvider == nil {
+		nftProvider = func(family knftables.Family, table string) (NftablesAPI, error) {
+			return NewRealNftables(family, table)
+		}
+	}
+
+	return &NftablesConfigurator{
+		cfg:              cfg,
+		NetworkNamespace: cfg.NetworkNamespace,
+		ruleBuilder:      builder.NewNftablesRuleBuilder(cfg),
+		nftProvider:      nftProvider,
+	}, nil
+}
+
+// separateV4V6 takes a comma-separated CIDR string and splits it into separate IPv4 and IPv6 ranges.
+// It also marks whether loopback IPs are included and handles wildcard ("*") case.
+func (cfg *NftablesConfigurator) separateV4V6(cidrList string) (NetworkRange, NetworkRange, error) {
+	if cidrList == "*" {
+		return NetworkRange{IsWildcard: true}, NetworkRange{IsWildcard: true}, nil
+	}
+	ipv6Ranges := NetworkRange{}
+	ipv4Ranges := NetworkRange{}
+	for _, ipRange := range split(cidrList) {
+		ipp, err := netip.ParsePrefix(ipRange)
+		if err != nil {
+			_, err = fmt.Fprintf(os.Stderr, "Ignoring error for bug compatibility with istio-nftables: %s\n", err.Error())
```

Can you give more information for why we ignore this error?